### DISCUSSION TITLE
Add a Version Selector for the Documentation

### DISCUSSION THREAD

**Discussion by arsen39** [rainbow-me/rainbowkit#1873] [Ideas]
I am a web developer and blockchain developer. I use RainbowKit on several commercial projects in our company because I really, really like it. I've encountered a problem with the release of new versions of RainbowKit; the ways of interacting with the library and the nuances of setting up the library change quite significantly. However, there are no resources to upgrade the library's version on each project, so we have to work with slightly outdated versions of the library on some projects. And when there's a need to slightly adjust the setup, there's no way to refer to the documentation since it only covers the most current version. And this is a problem. Especially when a new developer joins the project, and you need to onboard them.

Therefore, I would like to "earnestly" ask you to add a version selection feature for the documentation.

**Comment by 0xmkh** [comment]
@arsen39 Yeah we have some few version selector documentations. 

Here is the list:
- `1.3.x` versions: https://v1.rainbowkit.com
- `0.12.x` versions: https://v0.rainbowkit.com

**Reply by arsen39** [reply]
Thanks a lot! It is really helpful 🙂!
